dashboard/config/initializers/fast_localization.rb

```diff
@@ -1,7 +1,10 @@
 # Because loading YAML locales is super-slow, only load english yml locale files in development
 # To load all locales for testing, add "load_locales: true" to locals.yml config
-if Rails.env.development? && (!CDO.load_locales)
-  locale_paths = Dashboard::Application.paths['config/locales'].expanded.select{|x| x.include?('en')}
+
+if (Rails.env.test? or Rails.env.development?) && (!CDO.load_locales)
```

Comment:
I've never been able to get UI tests to run locally,  I take it this means our UI tests rely on having access to more locales?